examples/tv-app/linux/AppImpl.cpp

```diff
@@ -202,20 +201,19 @@ ApplicationLauncherResponse ApplicationLauncherImpl::LaunchApp(ApplicationLaunch
     ApplicationLauncherResponse response;
     const char * testData = "data";
     response.data         = (uint8_t *) testData;
-    response.status       = EMBER_ZCL_APPLICATION_LAUNCHER_STATUS_SUCCESS;
+    response.status       = (uint8_t) chip::app::Clusters::ApplicationLauncher::StatusEnum::kSuccess;
```

Comment:
to_underlying or use the right type in `ApplicationLauncherResponse`.

Reply:
Updated.